### ISSUE TITLE
Success/failure messages aren't read by screen reader

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#4865]
After pressing the save changes button the screen reader does not read the pop-up message that appears with if the changes are saved correctly or not.

**Comment by Floppy** [comment]
Need to put these in an ARIA live region: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Guides/Live_regions
